### ISSUE TITLE
Color Fill tools

### ISSUE THREAD

**Issue by OmegaRogue** [flxzt/rnote#328]
**Is your feature request related to a problem? Please describe.**
Filling in shapes manually takes alot of time that is wasted
**Describe the solution you'd like**
A tool to fill in shapes with a paintbucket and with a lasso selection

**Comment by devloberto** [comment]
My girlfriend and I were just looking for this feature as well 🙂

**Comment by ChepakiChepaki** [comment]
Yes pleaaase that would be so nice 3: Im so tired of coloring my little drawings by hand!
For the lasso selection tool, do you mean something like medibang's edged fill brush? Being able to choose both the colors of the outline and filling would be nice for a fill selection tool :>
